src/src/Helpers/Rules_calculate.h

```diff
@@ -108,14 +106,15 @@ class RulesCalculate_t {
 
   // operators
   // precedence   operators         associativity
-  // 3            !                 right to left
+  // 4            !                 right to left
+  // 3            ^                 left to right
   // 2            * / %             left to right
-  // 1            + - ^             left to right
+  // 1            + -               left to right

```

Comment:
Hmm that was an odd bug. Strange it wasn't discovered before.

Reply:
There was a difference in documentation between the .h and .cpp file. Assumed the most info was most accurate, but obviously something was lost here.